packages/tre/src/plugins/core/index.ts

```diff
@@ -178,6 +178,20 @@ async function handleEvent(event) {
       }
     }
 
+    const headers = new Headers();
+    headers.append("Content-Type", "application/json")
+    await ${BINDING_SERVICE_LOOPBACK}.fetch("http://localhost/core/log", {
+        method: "POST",
+        headers: headers,
+        body: JSON.stringify({
+            "status": response.status,
+            "statusText": response.statusText,
+            "method": event.request.method,
+            "url": response.url,
+            "time": Date.now(),
```

Comment:
So `Date.now()` in a worker only returns the time from the beginning of the request according to [this documentation](https://developers.cloudflare.com/workers/learning/security-model#step-1-disallow-timers-and-multi-threading). So we'd have to use separate timers if we want to support this feature. I could call `Date.now()` outside of `formatResponse` but thought it would be essentially the same.

Reply:
Ah I didn't realize that it was different workerd